controller/hybridgateway/converter/http_route.go

```diff
@@ -236,16 +370,26 @@ func (c *httpRouteConverter) translate(ctx context.Context, logger logr.Logger)
 				c.clusterDomain,
 			)
 			if err != nil {
-				// TODO: decide how to handle target creation errors in converter
-				// For now, skip this rule
+				log.Error(logger, err, "Failed to create KongTarget resources for rule, skipping rule",
+					"upstream", upstreamName,
+					"backendRefs", rule.BackendRefs,
+					"parentRef", pRef)
+				translationErrors = append(translationErrors, fmt.Errorf("failed to create KongTarget resources for upstream %s: %w", upstreamName, err))
 				continue
 			}
+			log.Debug(logger, "Successfully created KongTarget resources",
+				"upstream", upstreamName,
+				"targetCount", len(targets))
 			for _, tgt := range targets {
 				c.outputStore = append(c.outputStore, &tgt)
 			}
 
 			// Build the KongService resource.
 			serviceName := namegen.NewName(httpRouteName, cpRefName, utils.Hash32(rule.BackendRefs)).String()
+			log.Debug(logger, "Building KongService resource",
```

Comment:
Building logs are at Trace level for Plugins and PluginBindings: we may want to either change the others to Trace or change Plugin and PluginBinding build logs to Debug.
```suggestion
			log.Trace(logger, "Building KongService resource",
```